ets-demo/etsdemo/images/image_LICENSE.txt

```diff
@@ -0,0 +1,18 @@
+The icons are mostly derived work from other icons. As such they are
+licensed accordingly to the original license:
+
+Project                 License                   File
+----------------------------------------------------------------------------
+Nuvola                  LGPL                      image_LICENSE_Nuvola.txt
+
+Unless stated in this file, icons are the work of Enthought, and are
+released under a 3 clause BSD license.
+
+Files and orginal authors:
+----------------------------------------------------------------------------
+examples/tutorials/images:
+                                 next.png | Nuvola
+                         parent.png | Nuvola
+                       previous.png | Nuvola
+                         reload.png | Nuvola
+                            run.png | Nuvola
```

Comment:
Q : The run.png looks like the default "settings" button :/ not sure if that's what's expected or if it's just mislabeled as `run.png`

Reply:
I don't know... I just copied the files across without any changes to them.

Reply:
It looks like `run.png` file is used as an `ImageResource` by `etsdemo.app.DemoFileHandler` but i don't see it in the empty application so i don't know what's happening there.